src/Uno.UI/UI/Xaml/Shapes/Line.cs

```diff
@@ -1,11 +1,18 @@
-﻿using System;
+﻿#if !__IOS__ && !__MACOS__
+#define LEGACY_SHAPE_MEASURE
+#endif
```

Comment:
You repeat this on many files: it should be in the project or a `.targets` file.

Reply:
I don't see this a "configuration flag" that can be configured in the project/targets. It's really just a way to explicit the condition and make clear which code should be removed from the shared files once this refactor has been ported on other platforms.

The recommended way to do that would have been to copy those parts of code in all platforms specific files ...